test/bulk-info.test.js

```diff
@@ -102,7 +105,31 @@ describe('Test bulk info plugin', () => {
           return Number.isNaN(num) ? false : num === 2;
         }),
       }).run();
+
       assert.ok(sizeCheck, 'Wrong selection size displayed');
     }).timeout(IT_DEFAULT_TIMEOUT);
+
+    // Test case for non-Latin SharePoint item info
+    it(`Bulk info plugin handles non-Latin SharePoint item info in ${env} list view`, async () => {
```

Comment:
I would remove this test from the loop over both supported `env`s because it really only makes sense for sharepoint. Add it as a single test at the end instead.

Reply:
done!